primeqa/ir/dense/dpr_top/dpr/searcher.py

```diff
@@ -6,6 +6,7 @@
 import numpy as np
 import ujson as json
 import logging
+import re
```

Comment:
See if you move this `import re` at the top. 

I try to follow convention recommended by linters
1. Python built-in libraries 
2. 3rd party libraries
3. Custom code import